scripts/common/sonic-utilities-build/build.sh

```diff
@@ -17,8 +17,8 @@ sudo pip3 install buildimage/target/python-wheels/sonic_yang_models-1.0-py3-none
 sudo dpkg -i buildimage/target/debs/buster/libyang_1.0.73_amd64.deb
 sudo dpkg -i buildimage/target/debs/buster/libyang-cpp_1.0.73_amd64.deb
 sudo dpkg -i buildimage/target/debs/buster/python2-yang_1.0.73_amd64.deb
-sudo dpkg -i buildimage/target/debs/buster/libswsscommon_1.0.0_amd64.deb
 sudo dpkg -i buildimage/target/debs/buster/{libnl-3-200_*.deb,libnl-genl-3-200_*.deb,libnl-nf-3-200_*.deb,libnl-route-3-200_*.deb,libhiredis0.14_*.deb}
```

Comment:
Can you add a blank line and a comment above this stating that the group below is required for swsscommon?